webdriver/tests/support/fixtures.py

```diff
@@ -183,17 +183,14 @@ def end():
             _current_session.end()
             _current_session = None
 
-    def create_session(body):
+    def create_session(capabilities):
         global _current_session
         _session = webdriver.Session(configuration["host"],
                                      configuration["port"],
-                                     capabilities=None)
-        # TODO: merge in some capabilities from the confguration capabilities
-        # since these might be needed to start the browser
-        value = _session.send_command("POST", "session", body=body)
+                                     capabilities=overwrite_dictionary(configuration["capabilities"], capabilities))
```

Comment:
@carlosgcampos @andreastt FYI, I needed to change this to `capabilities={"alwaysMatch": caps}` to get it working locally. (Where caps is the result of overwrite_dictionary(...))

Reply:
hmm, tests are already passing {"alwaysMatch": caps} to new_session, you shouldn't need to add it again.

Reply:
Wait, not all the tests are doing the same thing, it seems new_session tests are passing {"capabilities": caps}. I'll check those and update the patch, thanks @mjzffr